addons/refuel/functions/fnc_takeNozzle.sqf

```diff
@@ -53,11 +53,22 @@ if (isNull _nozzle) then { // func is called on fuel truck
             _newNozzle attachTo [_unit, [-0.02,0.05,-0.12], "righthandmiddle1"];
             _unit setVariable [QGVAR(nozzle), _newNozzle, true];
 
-            if (_target isKindOf "AllVehicles") then {
-                // Currently ropeCreate requires its first parameter to be a real vehicle
-                private _rope = ropeCreate [_target, _endPosOffset, _newNozzle, [0, -0.20, 0.12], REFUEL_HOSE_LENGTH];
-                _newNozzle setVariable [QGVAR(rope), _rope, true];
+            private _ropeTarget = _target;
+            if (!(_target isKindOf "AllVehicles")) then {
+                private _helper = QGVAR(helper) createVehicle [0,0,0];
+                hideObjectGlobal _helper;
```

Comment:
Keep in mind this has to run on the server, there is an event in common that can handle the locality:

`[QEGVAR(common,hideObjectGlobal), [_helper, true]] call CBA_fnc_serverEvent;`